src/LibraryManagementSystem.java

```diff
@@ -11,6 +13,23 @@ public class LibraryManagementSystem {
     public static void main(String[] args) {
     	
     }
+
+	/*
+	  *This method takes a  value as a parameter (email, password or userIndex)
+	  *It checks whether there is any invalid entry.
+     */
+	public static  boolean checkPassword(String email,String password,int userIndex){
+		return users[userIndex][2].equals(email)&& users[userIndex][3].equals(password);
+	}
+
+	public  static void printCheckPassword(String email,String password,int userIndex){
+		String response="Login Successful";
```

Comment:
please use `getUserIndexByUserId(String userId)` this method in order to get user index